src/main/java/mate/jdbc/services/DriverServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package mate.jdbc.services;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private static DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).get();
```

Comment:
what if is null inside? Better to throw some exception like `DriverNotFoundException` or return `Optional` to keep your code style